tests/installation/partitioning.pm

```diff
@@ -21,10 +21,21 @@ sub run() {
 
     assert_screen 'partitioning-edit-proposal-button', 40;
 
+    if (check_screen ("install-from-iso-wrong-hdd")) {
```

Comment:
I suggest you use a multi-tag assert_screen in combination with `match_has_tag` here, e.g. see tests/installation/partitioning_lvm.pm for the use of `match_has_tag`